frb/builds/build_frbs.py

```diff
@@ -139,8 +139,8 @@ def frb_190102():
                         364.545 * units.pc / units.cm**3,
                         z_frb=0.2912) # Updated redshift
     # Error ellipse [REQUIRED]
-    frb190102.set_ee(0.16, 0.2, 0., 68.) # Statistical (Day+2020)
-    frb190102.set_ee(0.44, 0.5, 0., 68., stat=False) # Systematic
+    frb190102.set_ee(0.16, 0.2, 90., 68.) # Statistical (Day+2020)
+    frb190102.set_ee(0.44, 0.5, 90., 68., stat=False) # Systematic
```

Comment:
`a` should never be smaller than `b`.
We should probably even enforce that in code.
So something isn't right here.